samples/snippets/visualbasic/VS_Snippets_CLR_System/system.DateTime.ToString/vb/ToString1.vb

```diff
@@ -3,27 +3,26 @@ Option Strict On
 
 ' <Snippet1>
 Imports System.Globalization
-Imports System.Threading
 
 Module DateToStringExample
-   Public Sub Main()
-      Dim currentCulture As CultureInfo = Thread.CurrentThread.CurrentCulture
-      Dim exampleDate As Date = #05/01/2008 6:32:06PM#
-      
-      ' Display the date using the current (en-US) culture.
-      Console.WriteLine(exampleDate.ToString())
+    Public Sub Main()
+        Dim currentCulture As CultureInfo = CultureInfo.CurrentCulture
+        Dim exampleDate As Date = #05/01/2008 6:32:06PM#
 
-      ' Change the current culture to fr-FR and display the date.
-      Thread.CurrentThread.CurrentCulture = CultureInfo.CreateSpecificCulture("fr-FR")
-      Console.WriteLine(exampleDate.ToString())
+        ' Display the date using the current (en-US) culture.

```

Comment:
How do you know the current culture is `en-US`? The original code was using `fr-FR`